pkg/distributor/distributor.go

```diff
@@ -456,7 +456,7 @@ func newRingAndLifecycler(cfg RingConfig, instanceCount *atomic.Uint32, logger l
 	}
 
 	var delegate ring.BasicLifecyclerDelegate
-	delegate = ring.NewInstanceRegisterDelegate(ring.ACTIVE, ringNumTokens)
```

Comment:
Why these changes in distributor? Can you confirm they're not necessary for this PR?

Reply:
They are not needed for this PR, I just noticed that the way the number of tokens constant was being used was inconsistent.